specs/engage-digital_openapi3.yaml

```
@@ -5,7 +5,15 @@ info:
   title: RingCentral Engage Digital API
   version: '1.0'
 servers:
-  - url: 'https://<yourdomain>.api.engagement.dimelo.com/1.0'
+  - url: 'https://{account_name}.{platform_hostname}/1.0'
+    description: API server endpoint
+    variables:
+      account_name:
+        default: domainname

```

To make it extra clear that hyphens are allowed.

```suggestion
        default: domain-name
```